live-build/variants/internal-dcenter/package-lists/dcenter.list.chroot

```diff
@@ -24,8 +24,10 @@ libldap2-dev
 libsasl2-dev
 nfs-kernel-server
 openjdk-11-jdk-headless
-python3
-python3-dev
+python3.12
+python3.12-dbg
```

Comment:
We aren't using the `python3-dbg` package. i imagine we might not need the python3.12 version of it either

Reply:
@jtamagnan-delphix , removed this.